### PR TITLE
Update to latest strapi version in the disclaimer in the README

### PR DESCRIPTION
**Pull request by brunoocasali**
This PR is just a suggestion, and it also raises a question: "Should we update this version every time we have a new strapi v3 version?"

### CODE REVIEW

**Review by bidoubiwa — APPROVED** [review]
<3 Thanks a lot for spotting this

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ To understand Strapi and how to create an app, see [Strapi's documentation](http
 
 ## 🔧 Installation
 
-This package version works with the [v4 of Strapi](https://github.com/strapi/strapi/tree/v4.1.3). If you are using [Strapi v3](https://github.com/strapi/strapi/tree/v3.6.9), please refer to [this README](https://github.com/meilisearch/strapi-plugin-meilisearch/tree/v0.5.1).
+This package version works with the [v4 of Strapi](https://github.com/strapi/strapi/tree/v4.1.3). If you are using [Strapi v3](https://github.com/strapi/strapi/tree/v3.6.9), please refer to [this README](https://github.com/meilisearch/strapi-plugin-meilisearch/tree/v0.6.0-strapi-v3.0).
```
Comment (bidoubiwa) [inline README.md:50]:
```suggestion
This package version works with the [v4 of Strapi](https://github.com/strapi/strapi/tree/v4.1.3). If you are using [Strapi v3](https://github.com/strapi/strapi/tree/v3.6.9), please refer to [this README](https://github.com/meilisearch/strapi-plugin-meilisearch/tree/v3_main).
```

Comment (brunoocasali) [reply README.md:50]: Awesome! :)

**Review by bidoubiwa — APPROVED** [review]
bors merge

**Pull request merged**